### PR TITLE
Nice shape for main plugin file

### PR DESCRIPTION
**Pull request by szepeviktor**
- indentation
- require_once is a statement
- explicit func. arg. count in add_action

@mnelson4

### CODE REVIEW

File: printmyblog.php [hunk 1]
```diff
@@ -1,24 +1,25 @@
 <?php
+
 /**
  * @package PrintMyBlog
  * @version 1.0
+ *
+ * @wordpress-plugin
```
Comment (mnelson4) [inline printmyblog.php]: I'm unfamiliar with this tag. Can you point me to some documentation so I know what this is doing?

File: printmyblog.php [hunk 2]
```diff
@@ -1,24 +1,25 @@
...
+ * Plugin Name: Print My Blog
```
Comment (mnelson4) [inline printmyblog.php]: Oh, yes I suppose those asterixes are fine to add. Go figure!

File: printmyblog.php [hunk 3]
```diff
@@ -42,38 +43,39 @@ function pmb_already_active()
         <?php
     }
 
-    add_action('admin_notices', 'pmb_already_active', 1);
+    add_action('admin_notices', 'pmb_already_active', 1, 0);
```
Comment (mnelson4) [inline printmyblog.php]: I suppose this is more correct (we don't expect any parameters to be passed in, whereas the default is actually to assume 1 is being passed).

File: printmyblog.php [hunk 4]
```diff
@@ -100,13 +101,13 @@ function pmb_plugin_activation()
...
\ No newline at end of file
+    require_once PMB_INCLUDES_DIR . 'constants.php';
+    require_once PMB_INCLUDES_DIR . 'vendor/mnelson4/RestApiDetector/RestApiDetector.php';
+    require_once PMB_INCLUDES_DIR . 'vendor/mnelson4/RestApiDetector/RestApiDetectorError.php';
+    require_once PMB_INCLUDES_DIR . 'domain/PrintOptions.php';
+    require_once PMB_INCLUDES_DIR . 'domain/FrontendPrintSettings.php';
+    require_once PMB_TWINE_INCLUDES_DIR . 'controllers/BaseController.php';
+    require_once PMB_INCLUDES_DIR . 'controllers/PmbInit.php';
+
+    (new PrintMyBlog\controllers\PmbInit())->setHooks();
```
[the remark this answers is not in the document]
Comment (szepeviktor) [reply printmyblog.php]: 👍

File: printmyblog.php (hunk 2 above)
Comment (szepeviktor) [reply printmyblog.php]: Here you go https://developer.wordpress.org/plugins/plugin-basics/header-requirements/#header-fields

File: printmyblog.php (hunk 1 above)
Comment (szepeviktor) [reply printmyblog.php]: Here you go https://developer.wordpress.org/plugins/plugin-basics/header-requirements/#header-fields

File: printmyblog.php (hunk 3 above)
Comment (szepeviktor) [reply printmyblog.php]: Yes. You're right.

File: printmyblog.php (hunk 4 above)
[the remark this answers is not in the document]
Comment (szepeviktor) [reply printmyblog.php]: One less temporary variable!

**Comment by szepeviktor** [comment]
@mnelson4 Please consider merging it before conflict grow as grass. 🍏

**Comment by mnelson4** [comment]
This has been merged into master (squash merge) and you were given props in the plugin's changelog.
Thanks!

**Comment by szepeviktor** [comment]
You're welcome.

**Pull request closed without merging**